content/about/index.md

```diff
@@ -2,4 +2,19 @@
 title: About
 ---
 
-Example about page (index)
+The ICAT software takes data from large scientific facilities - such as particle accelerators - and catalogues and indexes it so scientists can find the data they need and share it with their team. A simple REST interface allows scientific instrument designers to quickly upload their data to the system for indexing and cataloging. TopCat, ICAT's easy to use web interface, allows scientists and collaborators to effortlessly search and download their data. ICAT's secure authentication system allows principal investigators to control their data and ensures users can only access data which they are allowed to access. (Mention DOIs here?)
+
+ICAT is in use at some of the world's largest scientific facilities (list some?) enabling access to millions of (pounds or dollars) worth of scientific results. The software is free & open-source and has been developed & maintained by an international collaboration for over 10 years.
+
+### What does ICAT do?
+For a more detailed explanation of what ICAT does and the actors, entities and workflows it models, see the [ICAT Model](about/icat_model.md) page.
```

Comment:
The hyperlink leads to a page that does not exist at the moment, shall I get rid of this paragraph and put it back when the page is created?